pageserver/src/metrics.rs

```diff
@@ -1325,18 +1395,24 @@ impl RemoteTimelineClientMetrics {
         }
     }
 
-    pub fn remote_physical_size_gauge(&self) -> UIntGauge {
+    pub(crate) fn remote_physical_size_gauge_set(&self, sz: u64) {
```

Comment:
Could we instead use `set_remote_physical_size`? Similarly for the `xxxxxxx_get()`

Reply:
Considered that, liked this better because it groups all the functions that operate on the same piece of data under a common prefix.

Reply:
Wondered about a OnceCell and exposing the PerTimelineRemotePhysicalSizeGauge, it does not work with the `last_set` field. Atomics could work, but ... is this worthy.

Reply:
Could we at least remove the `_gauge` which the `TimelineMetrics` does not have, maybe mention the `resident_` prefix over at `TimelineMetrics` for more symmetry?